lib/test/Assertion.js

```diff
@@ -74,14 +74,16 @@ module.exports = class Assertions {
             if (jsonValue !== undefined) {
                 if (Array.isArray(this.value)) {
                     for (const value of this.value) {
-                        match = this.evaluateRegexOrString(this.operator, value, jsonValue, ignoreCase);
+                        const localizedValue = (testSuite && testSuite.getLocalizedValue(value)) || value;
```

Comment:
The assertion should already have a link to the test suite via the interaction object (interaction.test.testSuite)

Reply:
We need a new copy of testSuite, because we set a new locale.

Reply:
fixed on https://github.com/bespoken/skill-testing-ml/pull/71/commits/01505d2339d24b8a68fa23fad8859dd519f0684c